l10n_co_geo/__openerp__.py

```diff
@@ -0,0 +1,120 @@
+# -*- encoding: utf-8 -*-
+##############################################################################
+#
+#    Odoo, Open Source Management Solution
+#    This module copyright (C) 2015 DevCO Colombia SAS
+#    (http://devco.co).
+#
+##############################################################################
+
+{
+    'name': 'Colombian Geodata',
+    'version': '1.0',
+    'author': 'DevCO Colombia SAS',
+    'maintainer': 'DevCO Colombia',
+    'website': 'http://devco.co',
+    'license': 'AGPL',
+    'category': 'Localization',
+    'summary': 'such as cities, departments and some improvements on displaying them.',
+    'description': """
+
+Colombian Geodata
+=================
+
+This module was written to extend the functionality of DevCO's Colombian Basic Accounting (l10n_co) to support colombian departments and municipalities with their corresponding official codes. It also aims to improve usability in adressfields and helps with disambiguation (ej. Armenia, Quindio vs. Armenia, Antioquia).
+
+Installation
+============
+
+To install this module, you need to:
+
+ * just install the module. Necessary additional modules are installed automatically.
+
+Configuration
+=============
+
+To configure this module, you need to:
+
+ * do nothing, as we have built in sensible defaults.
+
+Usage
+=====
+
+To use (test) this module, you need to:
+
+ * go to your database management page and install an instance with demo data
+ * play around on adress fields.
+
+For further information, please visit:
+
+ * [Professional Support (DevCO)](http://devco.co/)
+ * [Community Support (Colombian Odoo Community)](https://plus.google.com/communities/113251920989277948689)
+
+Known issues / Roadmap
+======================
+
+ * this module is considered production-ready and has no specific roadmap or improvement schedule.
+ * it might be planned to include ZIP codes, as they will be more widely adopted in colombia and open sourced by the MinTIC (Colombian Ministry of Information Technology, which is owner and copyright holder of the colombian ZIP Code Database)
+
+Credits
+=======
+
+Contributors
+------------
+
+* David Arnold <dar@devco.co>
+* Juan Pablo Arias <jpa@devco.co>
+
+Maintainer
+----------
+
+.. image:: https://www.devco.co/logo.png
+   :alt: DevCO Colombia SAS
+   :target: http://devco.co
+
+This module is maintained by DevCO Colombia SAS.
```

Comment:
Alright, I thought that one. Devco would figure as contributor on the module, only.